charts/redpanda/templates/certs.yaml

```diff
@@ -37,7 +37,7 @@ metadata:
       {{- end }}
 spec:
   dnsNames:
-      {{- if (empty $data.issuerRef) }}
+      {{- if or (empty $data.issuerRef) (dig "applyInternalDNSNames" false $data) }}
```

Comment:
true?

Reply:
i am sorry, what do you mean? is the logic not what you wanted?

Reply:
OH, ok , yes we do not want that to be default for the or statement, as it would place those entries and override current behavior thus changing expectations (breaking change). We would like to keep things as we have them and provide dials for changing behavior, like in this case.